src/evict/evict_lru.c

```diff
@@ -682,8 +700,7 @@ __evict_pass(WT_SESSION_IMPL *session)
 			 */
 			WT_STAT_FAST_CONN_INCR(session,
 			    cache_eviction_server_slept);
-			__wt_cond_wait(session,
-			    cache->evict_cond, WT_THOUSAND * WT_MAX(loop, 1));
+			__wt_cond_wait(session, cache->evict_cond, WT_THOUSAND);
```

Comment:
This turned out to be important.  If something was blocking eviction for a while (e.g., checkpoints), the loop counter could be large, and it would take a long time to recover.